libstorage/TableFactoryPrecompiled.cpp

```diff
@@ -127,14 +127,21 @@ bytes TableFactoryPrecompiled::call(
             result = e.errorCode();
         }
 
-        out = abi.abiIn("", u256(result));
-        if (g_BCOSConfig.version() < RC2_VERSION)
+        if (result == 1)
```

Comment:
This code is a little confuse, please add some comment here.

Reply:
ok